envoy/datadog_checks/envoy/parser.py

```diff
@@ -0,0 +1,58 @@
+from .errors import UnknownMetric
+from .metrics import METRIC_PREFIX, METRIC_TREE, METRICS
+
+
+def parse_metric(metric):
```

Comment:
can we add a docstring roughly describing what this function does?

Reply:
Done!